src/sentry/eventstore/models.py

```diff
@@ -29,6 +29,21 @@ def ref_func(x):
     return x.project_id or x.project.id
 
 
+class EventIdentifier:
+    """
+    Identifier for an event to be retrieved
+    """
+
+    __slots__ = ["project_id", "event_id"]
+
+    def __init__(self, project_id, event_id):
+        self.project_id = project_id
+        self.event_id = event_id
+
+    def to_event(self):
+        return Event(self.project_id, self.event_id)
```

Comment:
Please don't introduce a dependency of the Snuba filter to the Snuba event model (filter shouldn't know anything about the model). Eventstore should be the one creating the Event objects like it for all of the rest of the methods. I'd suggest making EventIdentifier a tuple (or a named tuple if you prefer) and move it out of the models.py file.